server/models/User.js

```diff
@@ -1,52 +1,64 @@
-const mongoose = require('mongoose')
+const mongoose = require('mongoose');
 const UserSchema = new mongoose.Schema({
-    fullName: {
+  fullName: [
```

Comment:
Why is it an array?

Reply:
Was mistake, will fix it in the next commit action.